smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/bindingTraits/HTTPResponseTraitWithoutHTTPPayload.kt

```diff
@@ -29,19 +28,16 @@ class HTTPResponseTraitWithoutHTTPPayload(
     val customizations: HTTPProtocolCustomizable,
 ) : HTTPResponseBindingRenderable {
     override fun render() {
-        val bodyMembers = responseBindings.filter { it.location == HttpBinding.Location.DOCUMENT }
-        val bodyMembersWithoutQueryTrait = bodyMembers
-            .filter { !it.member.hasTrait(HttpQueryTrait::class.java) }
-            .toMutableSet()
+        val bodyMembers = responseBindings.filter { it.location == HttpBinding.Location.DOCUMENT }.toSet()
```

Comment:
Simplified this logic since there is no need to separate members with the `@httpQuery` trait from other members.